lib/src/toggle-group/ToggleGroup.jsx

```diff
@@ -99,17 +130,28 @@ const ToggleGroup = styled.div`
     props.margin && typeof props.margin === "object" && props.margin.bottom ? spaces[props.margin.bottom] : ""};
   margin-left: ${(props) =>
     props.margin && typeof props.margin === "object" && props.margin.left ? spaces[props.margin.left] : ""};
+  width: fit-content;
```

Comment:
Instead of width: fit-content; this should work too with display: inline-flex;

Reply:
Changed by `display: inline-flex`.